### DISCUSSION TITLE
Errors when deploying

### DISCUSSION THREAD

**Discussion by the-faisalahmed** [alshedivat/al-folio#2569] [Q&A] [answered]
I've followed all of the steps in the recommended approach in the installation guide and keep getting this error. I tried deleting and remaking the repository twice but no luck. Not sure where I'm going wrong. This was done within github.

<img width="431" alt="Error_SC" src="https://github.com/user-attachments/assets/15ea12d7-8c37-457b-a5d8-c381b41ddb41">

**Comment by gshanr** [comment]
I have the exact issue after following the recommended steps.

**Comment by george-gca** [comment, marked as answer]
Check our [FAQ](https://github.com/alshedivat/al-folio/blob/master/FAQ.md#my-webpage-works-locally-but-after-deploying-it-is-not-displayed-correctly-css-and-js-are-not-loaded-properly-how-do-i-fix-that).

**Reply by gshanr** [reply]
Thanks @george-gca. It is fixed now. I did not name the repo correctly. 

@the-faisalahmed Please follow the instructions precisely. I ran into issues because of an incorrect repository name.

**Reply by TheFlyingImagination** [reply]
@george-gca beat me to it!

**Reply by the-faisalahmed** [reply]
@gshanr Below is the name I used for my repo when copying the template. Is that incorrect?

<img width="282" alt="gitname" src="https://github.com/user-attachments/assets/75f89b34-9a05-4747-b21b-58b92e35cbff">

**Reply by george-gca** [reply]
Did you read the FAQ section above?

**Reply by the-faisalahmed** [reply]
@george-gca Yes, and if I'm not mistaken it just directs me back to the recommendations page. I'm not sure if the issue is occurring when I commit the _config.yml change to the url.

**Reply by the-faisalahmed** [reply]
@george-gca I think I've found the problem in emptying the baseline url. Site works now. Thank you

**Reply by george-gca** [reply]
Actually it directed to this section in FAQ:

`My webpage works locally. But after deploying, it is not displayed correctly (CSS and JS are not loaded properly). How do I fix that?`

Which is exactly about emptying the `baseurl`.
